sphinx/source/docs/user_guide/source_examples/styling_glyph_selections_plotting_params.py

```diff
@@ -0,0 +1,18 @@
+from bokeh.io import output_file, show
+from bokeh.plotting import figure
+
+output_file("styling_selections.html")
+
+plot = figure(plot_width=400, plot_height=400, tools="tap", title="Select a circle")
+renderer = plot.circle([1, 2, 3, 4, 5], [2, 5, 8, 2, 7], size=50,
+
+                       # set visual properties for selected glyphs
+                       selection_color="firebrick",
+
+                       # set visual properties for non-selected glyphs
+                       nonselection_fill_alpha=0.2,
+                       nonselection_fill_color="blue",
+                       nonselection_line_color="firebrick",
+                       nonselection_line_alpha=1.0)
```

Comment:
That's off topic, but I wonder how far would you go with this type of indentation? I think that at this point it already looks ugly and I would prefer just simple by-one indentation.